doc/DeveloperGuide.md

```diff
@@ -88,6 +88,33 @@ Use case ends.
 > 3a1. AddressBook shows an error message <br>
   Use case resumes at step 2
 
+### Use case: Renaming existing tag
+
+**MSS**
+
+1. User requests to see all the existing tags
+2. AddressBook shows a list of all the tags
+3. User requests to rename a specific tag in the list
+4. AddressBook confirms the change with the user
+5. User confirms change
+6. AddressBook changes the tag name
+Use case ends.
+
+**Extensions**
+
+2a. The list is empty
+
+> Use case ends
+
+3a. The tag to rename is invalid
+
+> 3a1. AddressBook shows an error message
+  Use case resumes at step 2
+
+5a. User cancels change
```

Comment:
Here should be `4a.`

In use case extensions, the numbering is use to describe the scenarios that can happen preventing the user from going to the next step. 
So in this case, User cancels the change prevents it from going step 5.

4. AddressBook confirms the change with the user
`4a.` User cancels change
5. User confirms change